### PR TITLE
license docs

### PR DESCRIPTION
**Pull request by karann-msft**

### CODE REVIEW

**Comment by karann-msft** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/create-packages/Creating-a-Package.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/karann-license/docs/create-packages/Creating-a-Package.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/create-packages/creating-a-package?branch=pr-en-us-1168) | [Details](#user-content-docs%2Fcreate-packages%2FCreating-a-Package.md)
[docs/reference/nuspec.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/karann-license/docs/reference/nuspec.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/reference/nuspec?branch=pr-en-us-1168) |

<a name="docs%2Fcreate-packages%2FCreating-a-Package.md"></a>
### [docs/create-packages/Creating-a-Package.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/karann-license/docs/create-packages/Creating-a-Package.md)
 - **Line 60**: **[Warning]** Invalid file link:(~/reference/nuspec#license).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C18%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201811182355250278-1168%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=k0vr0TVdzd6adud2tOR48P6qaPmNSPgy0S5bFb%2BHEpY%3D&st=2018-11-18T23%3A51%3A29Z&se=2018-12-19T23%3A56%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/create-packages/Creating-a-Package.md [hunk 1]
```diff
@@ -56,7 +56,8 @@ Common optional properties:
 - Copyright information
 - A short description for the [Package Manager UI in Visual Studio](../tools/package-manager-ui.md)
 - A locale ID
-- Home page and license URLs
+- Project URL
+- License as an expression or file (`licenseUrl` is being deprecated, use the [`license` nuspec metadata element](../reference/nuspec#license))
```
Comment (skofman1) [inline docs/create-packages/Creating-a-Package.md]: spdx expression

File: docs/reference/nuspec.md [hunk 2]
```diff
@@ -74,7 +74,52 @@ A comma-separated list of the package creators using profile names on nuget.org.
 #### projectUrl
 A URL for the package's home page, often shown in UI displays as well as nuget.org. 
 #### licenseUrl
+> [!Important]
+> licenseUrl is being deprecated. Use license instead.
+
 A URL for the package's license, often shown in UI displays as well as nuget.org.
+#### license
+An SPDX license expression or path to a license file within the package, often shown in UI displays as well as nuget.org.
```
Comment (skofman1) [inline docs/reference/nuspec.md]: This https://github.com/NuGet/docs.microsoft.com-nuget/blob/e8b1a32f0cdfb454d1aea2e5633bd08b64b91143/docs/reference/nuspec.md#example-nuspec-files
still mentions LicenseUrl

Comment (karann-msft) [reply docs/reference/nuspec.md]: yeah, that's fixed in my branch now.

**Pull request merged**